build-system/tasks/runtime-test/runtime-test-base.js

```diff
@@ -222,7 +222,8 @@ class RuntimeTestConfig {
         'report-config': {lcovonly: {file: `lcov-${testType}.info`}},
       };
 
-      const plugin = [
+      const newBabelifyConfig = Object.assign({}, this.babelifyConfig);
```

Comment:
Can we modify the plugin array before calling `Object.assign()`?

Reply:
What do you mean exactly? Concating `plugin` array and `this.babelifyConfig['plugins']`?

Reply:
Yeah exactly, then using that new array in `Object.assign()`. Also I think you can use `this.babelifyConfig.plugins`